_apps/your-app-name.md

```diff
@@ -0,0 +1,57 @@
+---
+# A human-friendly name of your listing
+title: Auto-comment
+# A short description of what your app does
+description:  Comments on new issues and pull requests based on your configuration.
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: https://github.com/apps/auto-comment
+# Include a few screenshots that show your app in action
+screenshots:
+- https://avatars3.githubusercontent.com/in/14394?s=88&amp;v=4
+# The GitHub usernames of anyone who authored the app
+authors: [ boyney123 ]
+# The repository where the code is located
+repository: https://github.com/boyney123/auto-comment
+host: https://auto-comment.now.sh
+---
+
+<p align="center">
+  <img src="https://avatars3.githubusercontent.com/in/14394?s=88&amp;v=4" width="64">
+  <h3 align="center"><a href="https://github.com/marketplace/auto-comment">auto-comment</a></h3>
+  <p align="center">A GitHub App built with <a href="https://github.com/probot/probot">Probot</a> that comments on new issues and pull requests based on your configuration.<p>
+  <p align="center"><a href="https://travis-ci.org/boyney123/auto-comment"><img src="https://img.shields.io/travis/boyney123/auto-comment/master.svg" alt="Build Status"></a>
+    <a href="https://codecov.io/gh/boyney123/auto-comment/"><img src="https://img.shields.io/codecov/c/github/boyney123/auto-comment.svg" alt="Codecov"></a>
+    <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/License-MIT-yellow.svg" alt="MIT"></a>
+ 
+  </p>
+</p>
+
+![todo](./example.png)
```

Comment:
You need to upload this image somewhere, since we don't have a local version of the image file in this repo.